titus-api/src/main/java/com/netflix/titus/api/jobmanager/model/job/JobDescriptor.java

```diff
@@ -36,6 +41,13 @@
 @ClassInvariant(expr = "@asserts.notExceedsComputeResources(capacityGroup, container)", mode = VerifierMode.Strict)
 public class JobDescriptor<E extends JobDescriptor.JobDescriptorExt> {
 
+    private static final DisruptionBudget DEFAULT_DISRUPTION_BUDGET = DisruptionBudget.newBuilder()
+            .withDisruptionBudgetPolicy(SelfManagedDisruptionBudgetPolicy.newBuilder().build())
```

Comment:
This could be a constant.

Reply:
I agree. We should probably have a constants file for budgets with specific meanings.